### PR TITLE
OSDOCS#45846 - Updating CentOS references

### PR DESCRIPTION
**Pull request by jherrman**
Version(s):
4.17 and later

Issue:
https://issues.redhat.com/browse/CNV-45846

Link to docs preview:
https://82225--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/storage/virt-automatic-bootsource-updates.html
https://82225--ocpdocs-pr.netlify.app/openshift-rosa/latest/virt/storage/virt-automatic-bootsource-updates.html

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/virt-autoupdate-custom-bootsource.adoc
Comment (lyarwood) [inline modules/virt-autoupdate-custom-bootsource.adoc:41]: instancetype.kubevirt.io/default-preference: centos.stream9

File: modules/virt-autoupdate-custom-bootsource.adoc
Comment (lyarwood) [inline modules/virt-autoupdate-custom-bootsource.adoc:49]: url: docker://quay.io/containerdisks/centos-stream:9

**Comment by dominikholler** [comment]
This PR is changing text outside Virtualization, so it would need a review from an openshift person.
Only modules/virt-autoupdate-custom-bootsource.adoc and modules/virt-verify-status-bootsource-update.adoc  seems to be related to virtualization.
@lyarwood can you please help to review the virtualization related part?

File: modules/virt-autoupdate-custom-bootsource.adoc
Comment (lyarwood) [inline modules/virt-autoupdate-custom-bootsource.adoc:55]: managedDataSource: centos-stream9 <4>

File: modules/virt-verify-status-bootsource-update.adoc
Comment (lyarwood) [inline modules/virt-verify-status-bootsource-update.adoc:40]: managedDataSource: centos-stream9

File: modules/virt-autoupdate-custom-bootsource.adoc
Comment (lyarwood) [inline modules/virt-autoupdate-custom-bootsource.adoc:37]: name: centos-stream9-image-cron

**Comment by jherrman** [comment]
@dominikholler , if we don't want to meddle in the source code for non-virt OCP docs, I can revert the commit. Nevertheless, if the CentOS support changes are aimed to be OCP-wide, I'd say it's worth it to update the docs centrally.

On that note, do you have an idea who would be a good point of contact for the non-virt-related review?

**Review by lyarwood — COMMENTED** [review]
I'm not sure the mass `s/centos7/centos9/g` is valid here given there has never been such a version. I've commented on the CNV related changes and suggested using our CentOS Stream related preferences, containerdisks and datasources.

**Comment by jherrman** [comment]
> I'm not sure the mass `s/centos7/centos9/g` is valid here given there has never been such a version. I've commented on the CNV related changes and suggested using our CentOS Stream related preferences, containerdisks and datasources.

Fair point, I'll revert the non-virt changes. Thank you for the input!

**Comment by jherrman** [comment]
@lyarwood , thank you again for the review. Just to be on the safe side, when you have a few minutes, please take a look at the updated changelog and preview, and let me know if everything's looking copacetic:

Diff: https://github.com/openshift/openshift-docs/pull/82225/files
Preview: https://82225--ocpdocs-pr.netlify.app/openshift-dedicated/latest/virt/storage/virt-automatic-bootsource-updates.html

Cheers,
J.

**Review by snarayan-redhat — COMMENTED** [review]
LGTM! 
Reminder to squash the commits before merging.

File: modules/virt-verify-status-bootsource-update.adoc
Comment (lyarwood) [inline modules/virt-verify-status-bootsource-update.adoc:62]: user-defined-centos-stream9

**Review by lyarwood — COMMENTED** [review]
Apologies for the delay, lgtm aside from one name where I think you dropped the `user-defined` prefix by  mistake.

**Review by lyarwood — COMMENTED** [review]
/lgtm

**Comment by geetikakay** [comment]
https://82225--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/storage/virt-automatic-bootsource-updates.html LGTM

**Review by RoniKishner — APPROVED** [review]

**Pull request merged**
